website/docs/docs/building-a-dbt-project/hooks-operations.md

```diff
@@ -238,7 +238,7 @@ Full usage docs can for the `run-operation` command can be found [here](run-oper
 
 These examples from the community highlight some of the use-cases for hooks and operations!
 
-* [In-depth discussion of granting privileges using hooks and operations](https://discourse.getdbt.com/t/the-exact-grant-statements-we-use-in-a-dbt-project/430)
+* [In-depth discussion of granting privileges using hooks and operations, for older versions of dbt](https://discourse.getdbt.com/t/the-exact-grant-statements-we-use-in-a-dbt-project/430)
```

Comment:
```suggestion
* [In-depth discussion of granting privileges using hooks and operations, for older versions (i.e. prior to 1.2.0)  of dbt](https://discourse.getdbt.com/t/the-exact-grant-statements-we-use-in-a-dbt-project/430)
```

Reply:
Great call out, @dataders! Can we just rephrase a little?
```suggestion
* [In-depth discussion of granting privileges using hooks and operations for dbt versions prior to 1.2](https://discourse.getdbt.com/t/the-exact-grant-statements-we-use-in-a-dbt-project/430)
```
```suggestion
* [In-depth discussion of granting privileges using hooks and operations, for older versions of dbt](https://discourse.getdbt.com/t/the-exact-grant-statements-we-use-in-a-dbt-project/430)
```